packages/frontend/src/components/Explorer/VisualizationCardOptions/index.tsx

```diff
@@ -46,7 +46,14 @@ const VisualizationCardOptions: FC = memo(() => {
                     return {
                         text: 'Mixed',
                         icon: (
-                            <IconChartAreaLine size={20} color={Colors.GRAY1} />
+                            <IconChartAreaLine
+                                size={20}
+                                color={
+                                    disabled === true
```

Comment:
you can just have `disabled ? Colors.LIGHT_GRAY1 : Colors.GRAY1`

Reply:
ok sure